src/client/pages/BezwarenDetail/BezwarenDetail.tsx

```diff
@@ -47,7 +52,7 @@ const BezwarenDetail = () => {
             <p>We kunnen op dit moment geen gegevens tonen.</p>
           </Alert>
         )}
-        {!!bezwaar && (
+        {!!bezwaar ? (

```

Comment:
Waarom heb je er een Ternary van gemaakt ? 

De andere kant is `<></>`

Reply:
Liep tegen rare issues aan daarom even ternery van gemaakt. Zet het weer terug.